lib/app/modules/us_core_guidance/us_core_r4_capability_statement_sequence.rb

```diff
@@ -55,9 +55,43 @@ class UsCoreR4CapabilityStatementSequence < CapabilityStatementSequence
         * [DSTU2 Conformance Statement](https://www.hl7.org/fhir/DSTU2/conformance.html)
       )
 
-      test 'FHIR server capability states JSON support' do
+      PROFILES = {
```

Comment:
This is a little bit redundant with the data provided in app/helpers/validation.rb, and perhaps in other places.  I'm a little worried if these change we won't remember to update all the different places that these get referenced.  But this is a bigger problem then just this PR, so I'm good with leaving this as is.

Reply:
Yeah, we might want to come up with some general module-level structure for storing these kinds of module-wide knowledge/behaviors.